myuw/templates/handlebars/teaching/photo_list.html

```diff
@@ -2,103 +2,216 @@
-                		<td>{{ class }}</td>
-                        {{ #if linked_sections }}                        
-                        {{ #each linked_sections }}                            
-                            {{ #each sections }}
-                		    <td>{{ this }}</td>
-                            {{ /each }}                           
-                        {{ /each }}                        
-                        {{ /if }}
-                        <td>{{ email }}</td>
-                	</tr>
-                    {{ /each }}
-                    </tbody>
-                </table>
-
-
-                <!-- mobile view 
                 <div id="student_list">
-                    {{ #each sections.0.registrations }}
-                        <div id="student_{{ regid }}">
-                            <div style="float: left;">
-                                <img class="classlist-photo" src="/photo/{{ url_key }}">
-                            </div>
-                            <div style="float: left; padding-left: 10px; padding-bottom: 10px;">
+
+                    <ol id="student_sort">
+
+                        {{ #each sections.0.registrations }}
+                            <li id="student_{{ regid }}">
+
+                                <img class="classlist-photo myuw-photo-list-image" src="/photo/{{ url_key }}">
+
+                                <div class="myuw-photo-list-content">
                                 <strong>{{ surname }}, {{ name }}</strong><br />
-                                Student Number: {{ student_number }} <br />
-                                UW NetID: {{ netid }} <br />
-                                Credits: {{ credits }} <Br />
-                                Major: {{ #each majors }} {{ full_name }} {{ /each }}<br />
-                                Class: {{ class }} <br />
-                                Email: <a href="mailto:{{ email }}">{{ email }}</a> <br />
-                                {{ #if linked_sections }}
-                                {{ #each linked_sections }}
-                                    {{ ucfirst type }} Section: 
-                                    {{ #each sections }}
-                                        {{ this }}
-                                    {{ /each }}
-                                    <br />
-                                {{ /each }}
-                                {{ /if }}
-                            </div>
-                            <br style="clear: both;">
-                        </div>
-                    {{ /each }}
-                </div> -->
-        
+
+
+                                <span class="sr-only">Email:</span> <a href="mailto:{{ email }}">{{ email }}</a> <br />
+
+                                    <div class="myuw-photo-list-meta">
+                                        Student Number: {{ student_number }} <br />
```

Comment:
should we use two `&nbsp;`s here between label and data like we did on the profile? Not a big deal either way here.